tools/rvn/Parameters/CreateSetupPackageParameters.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Threading;
+using McMaster.Extensions.CommandLineUtils;
+using Raven.Server.Commercial;
+
+namespace rvn.Parameters;
+
+internal class CreateSetupPackageParameters
+{
+    public string SetupInfoPath;
+    public string PackageOutputPath;
+    public CommandLineApplication Command;
+    public string SetupMode;
```

Comment:
Why is that a string instead of `Raven.Server.Commercial.SetupMode` enum?

Reply:
@arekpalinski it's different setup mode
https://github.com/ravendb/ravendb/pull/13563/files/cce45d0e060fdccdc0ba288c722327179a5f82e2#diff-ad9cbee56ef20d0464113960dae85d9641aae42f0fa937b47e2e599ea421fcdeR71

Reply:
Okay. Still wouldn't be easier to deal with enum instead of string?

Reply:
@arekpalinski I agree...